policy/modules/system/logging.te

```diff
@@ -605,8 +605,6 @@ optional_policy(`
 
 optional_policy(`
 	udev_read_db(syslogd_t)
-	# for systemd-journal to read seat data from /run/udev/data
-	udev_read_runtime_files(syslogd_t)
```

Comment:
This isn't duplicate see discussion in #230

Reply:
If this is only for systemd-journal, there is alaredy such line (line 557) in init_systemd conditon block.